File: src/main/java/models/MultiEvent.java
package models;

public class MultiEvent {

    private Long id;

    private String multiEventName;

    private int year;


    public MultiEvent(String name, int year){
        this.multiEventName = multiEventName;
        this.year = year;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getMultiEventName() {
        return multiEventName;
    }

    public void setMultiEventName(String multiEventName) {
        this.multiEventName = multiEventName;
    }

    public int getYear() {
        return year;
    }

    public void setYear(int year) {
        this.year = year;
    }
}

File: frontend/archive_database/src/components/multiEvents/MultiEvent.js
import React from "react";

const MultiEvent = (props) => {

  if(!props.multiEvent){
    return null;
  }



  return(
    <div className = "single-multiEvent">
      <div className="details">
        <p>Name: {props.multiEvent.multiEventName}</p>
        <p>Year: {props.multiEvent.year}</p>
      </div>
    </div>
  )
}
export default MultiEvent;
